Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -99,6 +127,11 @@ ProjectPamphletViewModelOutputs {
     self.configDataProperty.value = (projectOrParam, refTag)
   }
 
+  fileprivate let parentViewControllerProperty = MutableProperty<UIViewController?>(nil)
+  public func parent(viewController: UIViewController?) {
```

Comment:
Just wondering if we need to capture the whole parent VC here, what we're really interested in is the `safeAreaInsets.top` later on. Could we not update the VM by overriding `viewSafeAreaInsetsDidChange()` in the places that we care about?